CHANGELOG.md

```diff
@@ -26,6 +26,11 @@ Dropping a requirement of a major version of a dependency is a new contract.
 ## [Unreleased]
 [Unreleased]: https://bitbucket.org/atlassian/jira-actions/branches/compare/master%0Drelease-3.3.0
 
+### Fixed - 2019-02-27
```

Comment:
Shouldn't it actually be put in `Added` section? We are adding new API which in this PR as far as I can see.

Reply:
`RichTextEditorTextArea` is new API, so we definitely `Added` contracts